tools/adjust-db/README.md

```diff
@@ -0,0 +1,23 @@
+# adjust-db tool
+This is a tool that should only be used for testing purposes.  
+It is intended as a collection of commands that perform small db modifications.
+
+
+## change-db-kind
+Changes DbKind of a DB described in config (cold or hot).  
+Example usage:
+`neard adjust-db change-db-kind --new-kind RPC change-cold`  
```

Comment:
Did you consider passing the db path in a separate argument rather than using the existing --home? And then perhaps you can bypass all the temperature-kind checks and just forcefully write the new kind. This could be a bit cleaner API and less clumsy in the scenario described below.

Reply:
I mean, the scenario is clumsy on purpose, to demonstrate everything this tool is not. Documenting happy case is boring.
I think the tool serves its purpose as is, with taking DB from --home + config. I don't see the need for perfection here at this moment, I just want this to work in my case :)

But if I were to do it all again, I would do it your way, with opening arbitrary db and bypassing temperature checks, but it is not super trivial.